.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": true,
+  "arrowParens": "always"
+}
```

Comment:
Is this file an oversight? I don't see any packages installed or npm scripts that would leverage this file. I'm certainly not opposed to adding it, although eslint already supports prettier formatting that can be installed using the ESLint ecosystem. See https://prettier.io/docs/en/integrating-with-linters.html